### PR TITLE
Product cards task solution

### PR DESCRIPTION
**Pull request by VitaliiOkhotskyi**
- [DEMO LINK](https://VitaliiOkhotskyi.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://VitaliiOkhotskyi.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,55 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet"
+      href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__content">
+        <div class="image"></div>
```
Comment (etojeDenys) [inline src/index.html:21]: instead of the empty div tag we can add the image using `img`

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,55 @@
...
+              <div class="rate__star"></div>
+            </div>
+
+            <div class="reviews">Reviews: 5</div>
+          </div>
+
+          <div class="info__price price">
+            <div class="price__text">Price:</div>
+
+            <div class="price__numbers">$2,199</div>
```
Comment (etojeDenys) [inline src/index.html:47]: let's use semantic tags for text

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,55 @@
...
+          </div>
+        </div>
+
+        <a href="#buy"
+        class="button"
+        data-qa="hover">
+        buy
+        </a>
```
Comment (etojeDenys) [inline src/index.html:55]:
```suggestion
        <a 
          href="#buy"
          class="button"
          data-qa="hover"
        >
          buy
        </a>
```

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,162 @@
...
+  width: 166px;
+  height: 40px;
+  border: 1px solid $blue-accent;
+  border-radius: 5px;
+  font-size: 14px;
+  line-height: 16px;
+  color: $white;
+  font-weight: 700;
+
+  text-decoration: none;
```
Comment (etojeDenys) [inline src/styles/index.scss:153]: move it to another file

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,162 @@
...
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__prod-code {
+    margin-top: 4px;
+
+    color: $secondary;
+    font-size: 10px;
+    font-weight: 400;
```
Comment (etojeDenys) [inline src/styles/index.scss:69]: redundant

**Review by etojeDenys — CHANGES_REQUESTED** [review]
let's rename your classNames cause you should follow BEM and have only two blocks: `card` and `stars`

File: src/index.html [hunk 6]
```diff
@@ -5,9 +5,59 @@
...
+    <link rel="stylesheet"
+      href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__content">
+        <img src="./images/imac.jpeg"
+        alt="iMac"
+        class="image">
```
Comment (polosanya) [inline src/index.html:23]:
```suggestion
        <img 
          src="./images/imac.jpeg"
          alt="iMac"
          class="image"
        >
```
Fix all cases
image

File: src/index.html [hunk 7]
```diff
@@ -5,9 +5,59 @@
...
+
+        <div class="info">
+          <h1 class="info__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+          <div class="info__prod-code">Product code: 195434</div>
```
Comment (polosanya) [inline src/index.html:28]: image

File: src/index.html [hunk 8]
```diff
@@ -5,9 +5,59 @@
...
+
+          <div class="info__rate star">
+            <div class="rate star__4">
+              <div class="rate__star"></div>
```
Comment (polosanya) [inline src/index.html:32]: Why do you need `rate` class? It should be class `stars`
```suggestion
            <div class="stars stars--4">
              <div class="stars__star"></div>
```
image

File: src/styles/card.scss [hunk 9]
```diff
@@ -0,0 +1,22 @@
+.card {
+  margin: 50px;
```
Comment (polosanya) [inline src/styles/card.scss:2]: image

**Review by polosanya — CHANGES_REQUESTED** [review]
Please pay attention to the task description and checklist. There are should be only 2 BEM blocks: `card` and `stars`. All other are just elements of the card

**Pull request closed without merging**